src/content/blog/2018-01-29-simplifying-interledger-the-graveyard-of-possible-protocol-features.md

```diff
@@ -1,11 +1,12 @@
 ---
-layout: ../../layouts/BlogLayout.astro
```

Comment:
layout field is no longer supported in Markdown collection entries, see https://docs.astro.build/en/guides/upgrade-to/v5/#breaking-changes-to-legacy-content-and-data-collections